airflow/providers/google/cloud/hooks/cloud_sql.py

```diff
@@ -500,7 +500,7 @@ def _download_sql_proxy_if_needed(self) -> None:
             )
         proxy_path_tmp = self.sql_proxy_path + ".tmp"
         self.log.info("Downloading cloud_sql_proxy from %s to %s", download_url, proxy_path_tmp)
-        response = requests.get(download_url, allow_redirects=True)
+        response = httpx.get(download_url, allow_redirects=True)
```

Comment:
`httpx` isn't listed in the google providers deps.

Should it be?

Reply:
It's a core dependency of Airlfow now. but you are right if it is installed for older version it should be.

Reply:
Oh yeah. And yeah, explicit is probably better than implicit.